### PR TITLE
Adding entity validator lib package

### PR DESCRIPTION
**Pull request by nicolasarnold12321**
This adds a library that can be referenced in other projects. eg. roadie-bot, backstage-entity-validator

### CODE REVIEW

**Review by Xantier — APPROVED** [review]

File: utils/roadie-backstage-entity-validator/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Backstage entity validator
```
Comment (Xantier) [inline utils/roadie-backstage-entity-validator/README.md:1]: This doesn't make sense anymore, so maybe rewrite this file?

Comment (nicolasarnold12321) [reply utils/roadie-backstage-entity-validator/README.md:1]: Sure thing

File: utils/roadie-backstage-entity-validator/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
```
Comment (Xantier) [inline utils/roadie-backstage-entity-validator/README.md:1]:
```suggestion
# Roadie Backstage entity validator
```

Comment (nicolasarnold12321) [reply utils/roadie-backstage-entity-validator/README.md:1]: Sure thing

**Review by Xantier — APPROVED** [review]

**Pull request merged**